src/components/Placeholder/Placeholder.tsx

```diff
@@ -0,0 +1,14 @@
+import { useEffect, useState } from 'react';
+import { MAX_RESULTS } from '../../constants/api';
+import Overview from '../Overview';
+import { StyledUl } from './styles';
+
+export default function Placeholder() {
+  const [items, setItems] = useState<JSX.Element[]>([]);
+
+  useEffect(() => {
+    setItems(Array.from({ length: MAX_RESULTS }, () => <Overview />));
```

Comment:
Can you explain why you need a state for this?

Reply:
Array.from is not very optimized function right? I wanted it to rerun only if MAX_RESULTS change.